mscore/mixer.cpp

```diff
@@ -71,15 +71,15 @@ char userRangeToReverb(double v) { return (char)qBound(0, (int)(v / 100.0 * 128.
 //---------------------------------------------------------
 
 Mixer::Mixer(QWidget* parent)
-   : QWidget(parent, Qt::Dialog),
+    : QDockWidget("Mixer", parent),
       showDetails(true),
       trackHolder(nullptr)
       {
       setupUi(this);
 
-      setObjectName("Mixer");
```

Comment:
Necessary to remove?

Reply:
Shoot, just saw it. Ignore me :)

Reply:
I thought  now it was done by the QDockWidget constructor already, however I just double-checked that the first param of that constructor is actually the dockable/floating window TITLE, so the setObjectName() seems still useful to have, for find-by-name related purposes.

Reply:
... however, I suspect this line in the .ui makes that call really redundant:

 <widget class="QDockWidget" name="Mixer">

Reply:
<widget class="QDockWidget" name="Mixer">